dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryClientFilter.java

```diff
@@ -71,6 +71,11 @@ public void init() {
             if (instrumenter != null) {
                 return;
             }
+            
+            if(openTelemetry.toString().equals(OpenTelemetry.noop().toString())){
```

Comment:
This doesn't feel like a good way to check whether we're using the noop `OpenTelemetry` object. In general `toString()` is not fully defined so there's no guarantee that it will be the same for two equal objects, or different for two different objects.

I suspect `openTelemetry.equals(OpenTelemetry.noop())` would work (though `OpenTelemetry.noop().equals(openTelemetry)` may not), or we should find a different way to check if we're enabled.

Reply:
In my experimenting openTelemetry.equals(OpenTelemetry.noop()) did not work. I think because openTelemetry is proxied via CDI.

Reply:
We found a way around this by creating `OpenTelemetryInfo` to hold both the `OpenTelemetry` object and whether or not telemetry is enabled.